### PR TITLE
 Added styles to login and register component

### LINKED ISSUE #7: FE: A user can signup/register

### PR DESCRIPTION
**Pull request by Junjpan**
__Current View__
image

image

__Past View__
image

### CODE REVIEW

File: client/src/compontents/register/Register.js [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
     return (
-        <div>
-            Register Form
-        </div>
+
+        <Form form="register">
+            <H2>Create an Account </H2>
+            <FormWrap >
+                <Label htmlFor="name" >Username:</Label><br />
+                <Input type="text" name="name" id="name" required></Input><br />
```
Comment (rusye) [inline client/src/compontents/register/Register.js]: You could probably clean up those labels and inputs by combing them like so:
```
<label>Username: <br />
   <input type="text" name="name" id="name" required />
</label>
```
I think you can close off that input by with the forward slash in the end like so `<input ..... />`

File: client/src/compontents/register/Register.js [hunk 2]
```diff
@@ -1,11 +1,48 @@
...
+                <Label htmlFor="password">Password:</Label><br />
+                <Input type="text" name="password" id="password" required></Input><br />
+                <Label htmlFor="email">Email:</Label><br />
+                <Input type="text" name="email" id="email" required></Input><br />
+                <Button type="Register">Sign Up</Button>
+            </FormWrap>
+            <hr />
```
Comment (rusye) [inline client/src/compontents/register/Register.js:41]: What is this `<hr />`?

File: client/src/App.js [hunk 3]
```diff
@@ -1,26 +1,60 @@
...
   const [status,setStatus]=useState(false);
+  const [newUser,setNewUser]=useState(false);
+
+  function registerForm(){
+    setNewUser(true)
+  }
+
+  function loginForm(){
+    setNewUser(false)
+  }
```
Comment (rusye) [inline client/src/App.js]: Could we compress this into one? Maybe something like `newUser ? setNewUser(false) : setNewUser(true)`

File: client/src/compontents/login/Login.js [hunk 4]
```diff
@@ -1,11 +1,48 @@
-import React,{useState} from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Form from './../../theme/form';
+import Label from './../../theme/label';
+import Input from './../../theme/input';
+import Button from './../../theme/button';
```
Comment (rusye) [inline client/src/compontents/login/Login.js]: Hmm could this be compressed into some like `import Form, Label, Input, Button from './../../theme'`

File: client/src/compontents/login/Login.js [hunk 5]
```diff
@@ -1,11 +1,48 @@
...
+    }
+    `
+
+    const FormWrap=styled.div`
+    margin-top:30px;
+    margin-left:25%;
+    `
+
+function Login(props) {
+    const [name, setName] = useState('');
```
Comment (rusye) [inline client/src/compontents/login/Login.js:25]: I believe we are using email and password to login.

File: client/src/compontents/login/Login.js [hunk 6]
```diff
@@ -1,11 +1,48 @@
...
     return (
-        <div>
-            Login Form
-        </div>
+
+            <Form background="white" >
+                <H2 >Login </H2>
+                <FormWrap>
+                <Label htmlFor="name">Email:</Label><br />
+                <Input type="text" name="name" id="name" required></Input><br />
```
Comment (rusye) [inline client/src/compontents/login/Login.js]: See my comment in Register.js components regarding combining label and input

**Review by rusye — CHANGES_REQUESTED** [review]

File: client/src/compontents/register/Register.js (hunk 2 above)
Comment (Junjpan) [reply client/src/compontents/register/Register.js:41]: it is a line

File: client/src/compontents/login/Login.js (hunk 5 above)
Comment (Junjpan) [reply client/src/compontents/login/Login.js:25]: Yes, but after login we still want to get the name information from the backend as well and pass to the app component. but I did forget to update the email info.

File: client/src/compontents/register/Register.js (hunk 1 above)
Comment (Junjpan) [reply client/src/compontents/register/Register.js]: I think it is all based on people's preference. but I can change that as you suggested

File: client/src/App.js (hunk 3 above)
Comment (Junjpan) [reply client/src/App.js]: yes, I can do that.

File: client/src/compontents/login/Login.js (hunk 4 above)
Comment (Junjpan) [reply client/src/compontents/login/Login.js]: good suggestion, will do that!

**Review by a-w-m — APPROVED** [review]

**Review by rusye — APPROVED** [review]

**Pull request merged**